WebNetSample/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Business.Abstract;
+using Microsoft.AspNetCore.Mvc;
+using WebNetSample.Entity.Concrete;
+
+namespace WebNetSample.WebNetMVC.Controllers
+{
+    public class CategoriesController : Controller
+    {
+        private readonly ICategoryService _categoryService;
+
+        public CategoriesController(ICategoryService categoryService)
+        {
+            _categoryService = categoryService;
+        }
+
+        // GET: /<controller>/
+        public IActionResult Index()
+        {
+            //List<Category> categories = (from category in _context.Categories.Take(10)
+            //select category).ToList();
+
+            List<Category> categories = (from category in _categoryService.GetList().Take(10)
```

Comment:
1. Well, for C# it's better to **always** use implicit type with `var`, let's always follow this practice.
2. I think that searching for particular amout of items should be fully moved to _Service_ layer. I mean, you can add an overloading to your `GetList` and pass there argument `amount`. Later this amount should be passed for _DAL_ as well. This is a huge benefit to performance, because you will filter amount of items on DB side, not in server side. Your implementation will upload to server all entities (1000 as an example) and will take only 10, however, 990 will not be used and will stay in memory.

Reply:
- For part1: Doesn't it cause a `boxing` problem? I didnt prefer to use just because of performance issues of `boxing and unboxing`. But maybe I remember wrong about `var`
- For part2: Actually, I haven't completed pagination yet. Because it is not included in Task 1. But I'll do it in Task 2 as you suggested, thank you. Or should I implement right now?

Reply:
1. No, of course not. Boxing/unboxing is not working here since you type will be predefined by compiler and no any casts are required.
2. Yes, let's do this. Just don't forget about it)